python/pyspark/sql/functions.py

```diff
@@ -132,6 +132,9 @@ def lit(col: Any) -> Column:
 
```

Comment:
Missing functions in Spark Connect are:

```
window_time
call_udf
days
window
hours
pandas_udf
sequence
bucket
years
session_window
format_number
unwrap_udt
udf
months
```

FYI @zhengruifeng

Reply:
thank you, 

the `Partition Transformation Functions` is tracked in https://issues.apache.org/jira/browse/SPARK-41503

I will add udf-related function after udf is supported